UdpHosts/GameServer/Entities/Character/CharacterEntity.cs

```diff
@@ -1425,6 +1436,23 @@ private ulong GetCurrentPermissionsValue()
         return result;
     }
 
+    public void EquipItemByGUID(int loadoutId, LoadoutSlotType slot, ulong guid)
+    {
+        Player.Inventory.EquipItemByGUID(loadoutId, slot, guid);
+        ApplyLoadout(CurrentLoadout);
+        
+    }
+    
+    public void EquipVisualBySdbId(uint loadoutId, LoadoutVisualType visualSlot, LoadoutSlotType slot, uint sdb_id)
+    {
+        
+        Player.Inventory.EquipVisualBySdbId(loadoutId, visualSlot, slot, sdb_id);
+        Player.CharacterEntity.CurrentLoadout.GliderID = sdb_id;
+        Player.NetChannels[ChannelType.ReliableGss].SendIAero(Character_ObserverView, this.EntityId);
```

Comment:
You are only sending to the local player here so other players won't get this keyframe. But you should not need to force send a keyframe here after correcting the static info calls above. So this line should just be removed.

Reply:
Removed